### ISSUE TITLE
Make devil antag immune to firball/fire in all forms

### ISSUE THREAD

**Issue by Lightning38088** [yogstation13/yogstation#1946]
in the times i've played devil i've been damaged in the first few stages of transformation but the fireball always getting me on fire or blows a limb off and it's weird since shouldn't the devil be immune to damage from his own fireball?

**Comment by AdamElTablawy** [comment]
Fireballs cause explosions.

Explosions damage people.

Not sure how we can do that aside from giving the devil bomb resistance/immunity to the blast which would be meh

**Comment by Lightning38088** [comment]
Yeah bomb immunity or resistance would be cool since they are dealing out powers which can give people fireballs so it'd be natural for a devil to be tricky and not be able to be damaged by fireballs.
